cvat/apps/engine/models.py

```diff
@@ -1091,9 +1093,16 @@ class TrackedShapeAttributeVal(AttributeVal):
     shape = models.ForeignKey(TrackedShape, on_delete=models.DO_NOTHING,
         related_name='attributes', related_query_name='attribute')
 
+
 class Profile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     rating = models.FloatField(default=0.0)
+    has_analytics_access = models.BooleanField(
+        _("has /analytics access"),
```

Comment:
Changed description.
Since this field not only gives access to endpoint, but also is used in UI to determine whether the user should see "Analytics" link, the old description is kind of misleading.